slack_sdk/web/async_client.py

```diff
@@ -83,7 +83,7 @@ async def admin_analytics_getFile(
         type: str,
         date: Optional[str] = None,
         metadata_only: Optional[bool] = None,
-        **kwargs
+        **kwargs,
```

Comment:
This is fine to have these changes but, out of curiosity, do you use some code formatter that does this? I don't believe that you manually added these commas.

Reply:
Right, these were added by the Python extension I have in my VSCode, it seems. But I kept it in there because it seemed better style to leave the trailing commas in there.

Reply:
Got it. Thanks!